src/content/sellers-guide/index.mdx

```diff
@@ -40,14 +40,21 @@ Initial proposal submissions: Closed on October 20, 2023.
         img_src="images/sellers/have-a-contract-image.svg"
         img_alt="Person standing amidst a series of charts and graphs."    
     >
-        ### [Confirm you’re registered in Symphony and administer your master contract](/sellers-guide/register-in-symphony/)  
-        Authorized employees must register in Symphony to complete administrative duties; check out the registration instructions to help you through the process.
+        ### [Register in FedConnect to manage your contract - FedConnect.net](https://www.fedconnect.net/FedConnect/Default.htm)
+        - Add “[CALM@unisonglobal.com](CALM@unisonglobal.com)” to your trusted address book so messages from your GSA Contracting team do not go to spam. 
 
-       ### [Administer your task orders](/sellers-guide/make-changes-contract) 
-        View and respond to solicitations/RFIs/RFPs, submit Q&A and proposals, and manage task orders.
+       ### [Register in the Vendor Support Center (VSC) to see offers](https://vsc.gsa.gov/vsc/app-registration) 
+       - All offers for orders against your OASIS+ master contract will be in GSA eBuy. 
+       - You must register your OASIS+ contracts in VSC prior to using eBuy. Register your contracts at [https://vsc.gsa.gov/vsc/app-registration](https://vsc.gsa.gov/vsc/app-registration). 
+       - After successful registration, you should be able to log in to GSA eBuy when ordering opens. You can log into eBuy at that time using this link: https://www.ebuy.gsa.gov/ebuy/ . 
+       - VSC registration will be open. Check your award letter for additional details and dates. 
 
-        ### [Report sales in the FAS Sales Reporting Portal (SRP)](/sellers-guide/reporting) 
-        Review reporting requirements and discover how to use the SRP.
+        ### [Register in the Sales Reporting Portal (SRP)](https://srp.fas.gsa.gov) 
+       - Register in SRP to report task order awards, report sales monthly (including “no sales” against awarded task orders), and remit collected Contract Access Fees (CAF) quarterly. [https://srp.fas.gsa.gov](https://srp.fas.gsa.gov) 
```

Comment:
We usually put link associate with text, e.g. `[Register in SRP]((https://srp.fas.gsa.gov) `, not sure if this meets the content style guide by wrapping link with link,  maybe @rosamundtgov will have more input regarding this.

Reply:
I will change that...thank you...